examples/sglang/components/kv_router.py

```
@@ -0,0 +1,304 @@
+        await kv_listener.create_service()
+
+        if self.router_type == RouterType.KV:
+            self.indexer = KvIndexer(kv_listener, self.args.block_size)
+        self.metrics_aggregator = KvMetricsAggregator(kv_listener)
+        logger.info("KV Router initialised (type=%s)", self.router_type)
+
+    # ------------------------------------------------------------------
+    # Internal helpers
+    # ------------------------------------------------------------------
+
+    def _cost_function(
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)